src/mixins/countries-provinces-mixin.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, Vue } from 'vue-property-decorator'
+
+import countries from 'country-list/data.json'
+import provinces from 'provinces/provinces.json'
+
+// import these and sort them only once globally
+window['countries'] = window['countries'] || countries
```

Comment:
This actually is a game changer. That makes my life so much easier.
![uncaught error requre](https://github.com/bcgov/bcrs-shared-components/assets/122301442/20b9284a-bc1f-4653-9971-696d7e1c4eb5)

I cannot wait for this PR to be merged 😁

Reply:
lol didn't take long